Sources/protoc-gen-swift/MessageGenerator.swift

```diff
@@ -250,61 +250,25 @@ class MessageGenerator {
     }
 
     private func generateProtoNameProviding(printer p: inout CodePrinter) {
-        if descriptor.reservedNames.isEmpty && descriptor.reservedRanges.isEmpty {
-            if fields.isEmpty {
-                p.print("\(visibility)static let _protobuf_nameMap = \(namer.swiftProtobufModulePrefix)_NameMap()")
-            } else {
-                p.print("\(visibility)static let _protobuf_nameMap: \(namer.swiftProtobufModulePrefix)_NameMap = [")
-                p.withIndentation { p in
-                    for f in fields {
-                        for n in f.fieldMapNames {
-                            p.print("\(f.number): \(n),")
-                        }
-                    }
-                }
-                p.print("]")
-            }
+        var writer = ProtoNameInstructionWriter()
+        for f in fields {
```

Comment:
This should probably be `fieldsSortedByNumber`, I'll bet we're getting some negative values with real world protos for the delta that pads things out to more bytes.

Reply:
Oops, I thought I had done that already... fixed.